core/dsql.go

```diff
@@ -83,10 +83,9 @@ func (q DSQLQuery) String() string {
 
 	// Where
 	if q.Where != nil {
-		whereClause := sqlparser.String(q.Where)
-		whereClause = strings.Trim(whereClause, "'")
-		whereClause = replacePlaceholders(whereClause)
-		parts = append(parts, fmt.Sprintf("WHERE '%s'", whereClause))
+		whereStr := sqlparser.String(q.Where)
+		whereStr = strings.Replace(whereStr, TempValue, CustomValue, -1)
+		parts = append(parts, fmt.Sprintf("WHERE %s", whereStr))
```

Comment:
Do we need this change here? It looks like we're not replacing the TempKey anymore, why?